frontend/src/metabase/visualizations/visualizations/CartesianChart/use-models-and-option.ts

```diff
@@ -114,6 +115,8 @@ export function useModelsAndOption({
       return {};
     }
 
+    const shouldAnimate = !isPlaceholder && !isCypressActive;
+
```

Comment:
Coping my comment from #43984: any chance we could use the `@prefers-reduced-motion` media setting for this? IIRC it's already disabled in Cypress, and I suppose people won't expect charts to be animated when the setting is on as well

Reply:
@kulyk great idea, I'll explore it and add to https://github.com/metabase/metabase/pull/43984